docs/dashboard/notifications.md

```diff
@@ -0,0 +1,118 @@
+<!--
+title: Serverless Dashboard - Notifications
+menuText: Notifications
+layout: Doc
+-->
+
+<!-- DOCS-SITE-LINK:START automatically generated  -->
+
+### [Read this on the main serverless docs site](https://www.serverless.com/framework/docs/)
+
+<!-- DOCS-SITE-LINK:END -->
+
+# Notifications
+
+The Serverless Framework can notify you in **Slack**, **Email**, **SNS Topics**, or via 
+**webhooks** when [alerts](./insights.md#alerts) are generated for your application. Multiple notifications can be added to an application, each of which can be configured differently for different alerts, stages or services.
+
+## Add a Notification to an Application
+
+1. Navigate to the **applications** tab from the main menu.
+2. Select the application for which you would like to configure the notifications and expand the application view.
+3. Open the **notifications** tab in the application.
+4. Follow the "**Click here to create your first notification.**" link.
+5. Fill out the form and click **add notification** to save the new notification. The notification can be scoped to include only the **alerts**, **stages** or **services** you want.
+    - **alerts** - select "all alerts" to be notified about all alerts, including alerts which may be made available in the future, or individually select the alerts. The [Alerts](./insights.md#alerts) section provides more details on each of the available alert types.
+    - **stages** - select "all stages" to be notified about alerts on all stages, including stages which may be created in the future, or individually select the stages. The stages must be [configured](./profiles.md#add-a-deployment-profile-to-your-application-and-stage) on the application first for them to be available.
+    - **services** - select "all services" to notified about alerts on all services, including services which will be deployed in the future, or individually select the services.
+
+
+## Configure an existing SNS Topic
+
+In order for Serverless Framework to notify you in a **SNS Topics**, it must be granted
+permission to publish to the topic. If you have an existing SNS Topic this is how you configure the Access Policy on the Topic to grant Serverless Framework permission to publish
+notifications to the topic.
+
+
+1. Go to the [SNS in the AWS Console](https://console.aws.amazon.com/sns/v3/home).
+2. Select the desired topic from the topics list.
+3. When adding a notification to an application, use the ARN of the ARN in the **SNS Topic ARN** field.
```

Comment:
"use the ARN of the ARN in the **SNS Topic ARN** field"

Seems like a Typo. Should be "use the ARN of the Topic in the **SNS Topic ARN** field"